### ISSUE TITLE
[feature request] Distance in the note list

### ISSUE THREAD

**Issue by opk12** [hauke96/GeoNotes#37]
I collect points of interest on the phone as notes, during various trips across the same areas. At a later time, I upload them in an OpenStreetMap editor at home.

But OSM prefers geographically small changesets for technical reasons. So my typical query is "all the notes not far from this point" that are surveyed and ready to be upstreamed (#21) and contain the text "hydrant" (#34).

**Comment by opk12** [comment]
I understand that a complex query mechanism may be too much for this app. If so, at least it would be convenient for me to
- sort notes by distance
- when sorted by other criteria, read the distance next to the note

**Comment by hauke96** [comment]
My question would be: Distance to what? The current center of the map? A selected point/coordinate on the map?

Maybe a general filter would be enough? I can imagine the following: 
When I filter the notes in the list (#34) then the filter also applies to all notes visible on the map. So when I search for "hydrant" all notes on the map then contain the word "hydrant" and I easily can see nearby hydrant-notes.

**Comment by opk12** [comment]
Yes, applying the search filter to the slippy map would do it for me.
